src/components/TalentsListPage/TalentsListPage.jsx

```diff
@@ -1,6 +1,14 @@
 import React from "react";
 import { Layout } from "../Layout";
-
+import { SearchPanel } from "../TalentsListPage/SearchPanel/SearchPanel";
```

Comment:
```suggestion
import { SearchPanel } from "../TalentsListPage/SearchPanel";
```

Reply:
maybe you wanted to say:
```suggestion
import { SearchPanel } from "./SearchPanel";
```